### DISCUSSION TITLE
[A-L] (2023/24) Foglio 15 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#288] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/dbb48376-2869-436a-b4d2-0debcab140ba)

**Comment by Elia-Belli** [comment]
Trovando una base per $Ker A=U$ possiamo determinare le equazioni parametriche per $U$: ci basta risolvere $Ax=0$ passando al sistema a scala equivalente $Sx=0$

$$S=\begin{pmatrix}1&1&1&-2\\
0&2&-2&-6\\
0&0&0&0\end{pmatrix}\to \begin{cases}x_1=-x_2-x_3+2x_4\\
x_2=x_3+3x_4\\
x_3=x_3\\
x_4=x_4\end{cases}\iff\begin{cases}x_1=-2x_3-x_4\\
x_2=x_3+3x_4\\
x_3=x_3\\
x_4=x_4\end{cases}$$

$$\Rightarrow KerS=x_3(-2,1,1,0)+x_4(-1,3,0,1)=Ker A$$

> Ho scritto i vettori in riga per comodità

Per passare alle equazioni cartesiane risolvo il seguente sistema e impongo compatibilità:

$$\begin{cases}-2w_1-w_2=x_1\\
w_1+3w_2=x_2\\
w_1=x_3\\
w_2=x_4\end{cases}$$

$$\begin{pmatrix}-2&-1\\
1&3\\
1&0\\
0&1\end{pmatrix}\begin{pmatrix}x_1\\
x_2\\
x_3\\
x_4\end{pmatrix}\to^{(Gauss)}\begin{pmatrix}1&0\\
0&1\\
0&0\\
0&0\end{pmatrix}\begin{pmatrix}x_3\\
x_4\\
x_1+2x_3+x_4\\
x_2-x_3-3x_4\end{pmatrix}$$

$$U=\\{\underline x\in \mathbb{R}^4|\begin{cases}x_1+2x_3+x_4=0\\
x_2-x_3-3x_4=0\end{cases}\\}$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/494c9f6e-bb82-4aca-9b9b-240f921e66e4)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/ac7fd3d3-2f6b-40e9-9ce5-bfd928c3d67f)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/18007d34-1584-4c96-ae81-ffbcb4b39d38)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-25 171811](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/d6dcdf9b-82cf-4ccf-bd71-4b71df707086)
